app/src/main/java/com/sdsmdg/harjot/MusicDNA/Activities/HomeActivity.java

```diff
@@ -78,13 +79,18 @@
 import com.sdsmdg.harjot.MusicDNA.CustomBottomSheetDialogs.CustomLocalBottomSheetDialog;
 import com.sdsmdg.harjot.MusicDNA.CustomViews.CustomLinearGradient;
 import com.sdsmdg.harjot.MusicDNA.Fragments.AboutFragment.AboutFragment;
+import com.sdsmdg.harjot.MusicDNA.Fragments.AllFoldersFragment.FolderFragment;
 import com.sdsmdg.harjot.MusicDNA.Fragments.AllPlaylistsFragment.AllPlaylistsFragment;
 import com.sdsmdg.harjot.MusicDNA.Fragments.EditSongFragment.EditLocalSongFragment;
 import com.sdsmdg.harjot.MusicDNA.Fragments.EqualizerFragment.EqualizerFragment;
 import com.sdsmdg.harjot.MusicDNA.Fragments.FavouritesFragment.FavouritesFragment;
 import com.sdsmdg.harjot.MusicDNA.Fragments.FolderContentFragment.FolderContentFragment;
-import com.sdsmdg.harjot.MusicDNA.Fragments.AllFoldersFragment.FolderFragment;
+import com.sdsmdg.harjot.MusicDNA.Fragments.LocalMusicFragments.AlbumFragment;
+import com.sdsmdg.harjot.MusicDNA.Fragments.LocalMusicFragments.ArtistFragment;
+import com.sdsmdg.harjot.MusicDNA.Fragments.LocalMusicFragments.LocalMusicFragment;
+import com.sdsmdg.harjot.MusicDNA.Fragments.LocalMusicFragments.LocalMusicViewPagerFragment;

```

Comment:
Oh I didn't realize that. I only added the haptic feedback lines of code. It happened by itself, I think. I will fix it.